components/head.js

```diff
@@ -22,6 +22,7 @@ const Head = props => (
       <link rel='apple-touch-icon' href='/static/touch-icon.png' />
       <link rel='mask-icon' href='/static/favicon-mask.svg' color='#49B882' />
       <link rel='icon' href='/static/favicon.ico' />
+      <meta property='og:type' content='website' />
       <meta property='og:url' content={props.url || defaultOGURL} />
```

Comment:
let's add 
`<meta property='og:image' content={defaultOGImage} />`

Reply:
that tag exists on [line 35](https://github.com/DhyeyLalseta/TeacherFund_next/blob/3b070c2fc86551c6f85ee1d4ce83e2809b118a62/components/head.js#L35) of the file , but it's conditional with props.ogImage, should it only use the default OG image?

Reply:
sorry i didn't see that!!